front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -32,216 +12,38 @@ export function ConversationTitle({
-  const [editedTitle, setEditedTitle] = useState<string>("");
-
-  const titleInputFocused = useRef(false);
-  const saveButtonFocused = useRef(false);
-
-  const handleClick = useCallback(async () => {
-    await navigator.clipboard.writeText(shareLink || "");
-    setCopyLinkSuccess(true);
-    setTimeout(() => {
-      setCopyLinkSuccess(false);
-    }, 1000);
-  }, [shareLink]);
-
-  const onTitleChange = useCallback(
-    async (title: string) => {
-      try {
-        const res = await fetch(
-          `/api/w/${owner.sId}/assistant/conversations/${activeConversationId}`,
-          {
-            method: "PATCH",
-            headers: {
-              "Content-Type": "application/json",
-            },
-            body: JSON.stringify({ title }),
-          }
-        );
-        await mutate(
-          `/api/w/${owner.sId}/assistant/conversations/${activeConversationId}`
-        );
-        void mutate(`/api/w/${owner.sId}/assistant/conversations`);
-        if (!res.ok) {
-          throw new Error("Failed to update title");
-        }
-        setIsEditingTitle(false);
-        setEditedTitle("");
-      } catch (e) {
-        alert("Failed to update title");
-      }
-    },
-    [activeConversationId, mutate, owner.sId]
-  );
-
   if (!activeConversationId) {
     return null;
   }
 
   return (
-    <>
-      <DeleteConversationsDialog
-        isOpen={showDeleteDialog}
-        type="selection"
-        selectedCount={1}
-        onClose={() => setShowDeleteDialog(false)}
-        onDelete={() => {
-          setShowDeleteDialog(false);
-          void onDelete();
-        }}
-      />
-      <AppLayoutTitle>
-        <div className="grid h-full min-w-0 max-w-full grid-cols-[1fr,auto] items-center gap-4">
-          <div className="flex min-w-0 flex-row items-center gap-4 text-primary dark:text-primary-night">
-            {!isEditingTitle ? (
-              <div className="dd-privacy-mask min-w-0 overflow-hidden truncate text-sm font-normal">
-                {conversation?.title || ""}
-              </div>
-            ) : (
-              <div className="w-[84%]">
-                <Input
-                  value={editedTitle}
-                  onChange={(e) => setEditedTitle(e.target.value)}
-                  onFocus={() => (titleInputFocused.current = true)}
-                  onBlur={() => {
-                    setTimeout(() => {
-                      if (!saveButtonFocused.current) {
-                        setIsEditingTitle(false);
-                      }
-                      titleInputFocused.current = false;
-                    }, 0);
-                  }}
-                  onKeyDown={(e) => {
-                    if (e.key === "Enter") {
-                      return onTitleChange(editedTitle);
-                    }
-                  }}
-                  autoFocus
-                />
-              </div>
-            )}
-
-            {isEditingTitle ? (
```

Comment:
shouldn't we keep the in-place conversation title edit ?

Reply:
As seen with ed: we should keep only a modal